storm-core/src/clj/backtype/storm/command/rebalance.clj

```diff
@@ -14,19 +14,20 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.command.rebalance
-  (:use [clojure.tools.cli :only [cli]])
-  (:use [backtype.storm thrift config log])
+  (:require [clojure.tools.cli :refer [cli]]
+            [backtype.storm.thrift :as thrift]
+            [backtype.storm.log :as log])
   (:import [backtype.storm.generated RebalanceOptions])
   (:gen-class))
+;; TODO: do we need all these gen-class's?
```

Comment:
For the commands, yes. These are the classes launched by bin/storm.
If you find other `:gen-class` instances you think we can get rid of, feel free to point them out.